functions.cmake

```diff
@@ -549,9 +550,24 @@ function(install_java_artifact NAME)
 endfunction()
 
 function(install_java_artifact_dependencies NAME)
-    install(DIRECTORY "target/dependency/" DESTINATION lib/jars FILES_MATCHING PATTERN "*.jar")
+    install(DIRECTORY "target/dependency/" DESTINATION lib/jars/${NAME} FILES_MATCHING PATTERN "*.jar")
+    install(CODE "execute_process(COMMAND bash -c \"cd \$ENV{DESTDIR}/${CMAKE_INSTALL_PREFIX}/lib/jars && if [ -d ${NAME} ]; then ln -sf ${NAME}/*.jar . ; fi\")")
 endfunction()
 
 function(install_fat_java_artifact NAME)
     install(FILES "target/${NAME}-jar-with-dependencies.jar" DESTINATION lib/jars/)
 endfunction()
+
+function(install_symlink TARGET LINK)
+    install(CODE "execute_process(COMMAND ln -sf ${CMAKE_INSTALL_PREFIX}/${TARGET} \$ENV{DESTDIR}/${CMAKE_INSTALL_PREFIX}/${LINK})")
+endfunction(install_symlink)
+
+function(add_extra_projects)
+    if(EXTRA_PROJECTS)
```

Comment:
Consider adding "VESPA_" prefix to environment variable?

Reply:
I considered it, but the other EXTRA_ stuff does not have it. Like EXTRA_INCLUDE_DIRECTORY, EXTRA_LINK_DIRECTORY etc.

Reply:
I see. LGTM